_api-reference/document-apis/bulk-streaming.md

```diff
@@ -0,0 +1,81 @@
+
+The default HTTP transport method does not support streaming. You must install the [`transport-reactor-netty4`]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/network-settings/#selecting-the-transport) HTTP transport plugin and use it as the default HTTP transport layer. Both the `transport-reactor-netty4` plugin and the Streaming Bulk API are experimental.
+{: .note}
+
+## Path and HTTP methods
+
+```json
+POST _bulk/stream
+POST <index>/_bulk/stream
+```
+
+If you specify the index in the path, then you don't need to include it in the [request body chunks]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/#request-body).
+
+OpenSearch also accepts PUT requests to the `_bulk/stream` path, but we highly recommend using POST. The accepted usage of PUT---adding or replacing a single resource at a given path---doesn't make sense for streaming bulk requests.
+{: .note }
+
+
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+Parameter | Data type | Description
+:--- | :--- | :---
+`pipeline` | String | The pipeline ID for preprocessing documents.
+`refresh` | Enum | Whether to refresh the affected shards after performing the indexing operations. Default is `false`. `true` makes the changes show up in search results immediately, but hurts cluster performance. `wait_for` waits for a refresh. Requests take longer to return, but cluster performance doesn't suffer.
+`require_alias` | Boolean | Set to `true` to require that all actions target an index alias rather than an index. Default is `false`.
+`routing` | String | Routes the request to the specified shard.
+`timeout` | Time | How long to wait for the request to return. Default `1m`.
+`type` | String | (Deprecated) The default document type for documents that don't specify a type. Default is `_doc`. We highly recommend ignoring this parameter and using a type of `_doc` for all indexes.
+`wait_for_active_shards` | String | Specifies the number of active shards that must be available before OpenSearch processes the bulk request. Default is 1 (only the primary shard). Set to `all` or a positive integer. Values greater than 1 require replicas. For example, if you specify a value of 3, the index must have two replicas distributed across two additional nodes for the request to succeed.
+`batch_interval` | Time | Specifies how long bulk operations should be accumulated into batch before sending over to data nodes.
+`batch_size` | Time | Specifies how many bulk operations should be accumulated into batch before sending over to data nodes. Default `1`.
+{% comment %}_source | List | asdf
```

Comment:
Should this and the following two rows be here?

Reply:
They should (by API specs)